src/services/auth-services.ts

```diff
@@ -63,9 +63,9 @@ export default class AuthServices {
       if (response?.data) {
         return {
           contact: {
-            email: response.data.contacts[0].email,
-            phone: response.data.contacts[0].phone,
-            extension: response.data.contacts[0].phoneExtension
+            email: response.data.contacts[0]?.email || '',
+            phone: response.data.contacts[0]?.phone || '',
+            extension: response.data.contacts[0]?.phoneExtension || ''
```

Comment:
As I work through my ticket, I've done almost the same thing here!

![image](https://user-images.githubusercontent.com/10002889/170354846-2ccd570a-1068-413c-be18-a015e9c45cd8.png)

I found that I didn't need to assign empty strings (undefined is OK). What do you think?

I'll let you commit this change but please copy over my comment.

Reply:
Sure.